packages/calcite-components/src/components/dropdown-group/dropdown-group.tsx

```diff
@@ -43,10 +43,10 @@ export class DropdownGroup {
   @Prop() scale: Scale = "m";
 
   /**
-   * Specifies the selection mode for `calcite-dropdown-item` children, defaults to `single`:
-   * - `multiple` allows any number of selected items,
-   * - `single` allows only one selection (default),
-   * - `none` doesn't allow for any selection.
+   * Specifies the selection mode for `calcite-dropdown-item` children:
+   * - `"multiple"` allows any number of selected items,
+   * - `"single"` allows only one selection,
+   * - `"none"` doesn't allow for any selection.
```

Comment:
Similar to https://github.com/Esri/calcite-design-system/pull/8536#pullrequestreview-1801192987 - nitpick: _Believe_ in this case both the `:` and `-` / `,` and `-` would be shown together. Maybe remove the `-` throughout so only one character is displayed in the doc?